src/test/java/core/basesyntax/service/impl/OperationProcessImplTest.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.OperationProcess;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class OperationProcessImplTest {
+    private OperationProcess operationProcess;
+    private List<FruitTransaction> handledTransactions;
+
+    @BeforeEach
+    void setUp() {
+        OperationHandler operationHandler = fruitTransaction
+                -> handledTransactions.add(fruitTransaction);

```

Comment:
You shouldn't provide some new extra implementation of operationHandler in tests, let the use the existing ones